### PR TITLE
Adding deprecation message for old templates

### PR DESCRIPTION
**Pull request by tobiasqueck**

### CODE REVIEW

**Review by devinea — APPROVED** [review]
changes seem appropriate

File: packages/fiori-freestyle-writer/README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # @sap-ux/fiori-freestyle-writer
 
-Easy to use Fiori Freestyle project files writer for use within Yeoman generator and other prompting libraries. 
+Writer module allowing to generate custom SAPUI5 applications with different templates.
 
+:warning: **All templates except the Basic template have been deprectated** and will be removed at some point in time. Please use the Custom page SAP Fiori template (`@sap-ux/fiori-elements-writer`)as an alternative.
```
Comment (theStiggySpd) [inline packages/fiori-freestyle-writer/README.md:5]: Please update the following:
- Spell check `deprectated`
- Change `at some point in time` to `by the end of 2023`

**Review by theStiggySpd — CHANGES_REQUESTED** [review]
Couple of comments

File: packages/fiori-freestyle-writer/README.md (hunk 1 above)
Comment (tobiasqueck) [reply packages/fiori-freestyle-writer/README.md]: Type fixed, text changed to match the message in the Fiori tools word by word. No, I won't add the specific date.

Comment (theStiggySpd) [reply packages/fiori-freestyle-writer/README.md]: thanks. `in a future release` sounds better :-)

**Review by theStiggySpd — APPROVED** [review]
Updated deprecated message looks good from my side

**Review by devinea — APPROVED** [review]
updates lg.

**Pull request merged**